.github/workflows/ci.yml

```diff
@@ -505,7 +508,7 @@ jobs:
         run: cargo test --no-run --lib --features full
         working-directory: tokio
         env:
-          RUSTFLAGS: --cfg loom --cfg tokio_unstable -Dwarnings
+          RUSTFLAGS: --cfg loom --cfg tokio_unstable --cfg tokio_taskdump -Dwarnings
```

Comment:
You don't actually set this when you run loom tests, so I wouldn't set it when compiling them either.